python/observability-online-eval/agno/dockerfile

```
@@ -0,0 +1,26 @@
+# Use an official Python image
+FROM python:3.10-slim
+
+# Install system dependencies (if needed)
+RUN apt-get update && apt-get install -y build-essential curl
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)